modin/engines/ray/generic/io.py

```diff
@@ -26,18 +26,26 @@ def file_exists(file_path):
     if isinstance(file_path, str):
         match = S3_ADDRESS_REGEX.search(file_path)
         if match:
-            s3fs.exists(file_path)
+            return s3fs.exists(file_path)
     return os.path.exists(file_path)
 
 
-def open_file(file_path, mode="rb"):
+def file_open(file_path, mode="rb"):
```

Comment:
We should move the `import s3fs` statement inside the functions that use it so it's not a top-level dependency.

Reply:
fixed. fancy default args to keep using only a singleton per instance